misc/python/materialize/mzcompose/__init__.py

```diff
@@ -111,11 +111,17 @@ def get_default_system_parameters(
         "persist_batch_columnar_format": (
             "both_v2" if version >= MzVersion.parse_mz("v0.112.0-dev") else "row"
         ),
-        "persist_batch_columnar_format_percent": "10",
+        "persist_batch_columnar_format_percent": "100",
         "persist_batch_delete_enabled": "true",
         "persist_batch_record_run_meta": (
             "true" if version >= MzVersion.parse_mz("v0.115.0-dev") else "false"
         ),
+        "persist_batch_structured_order": (
+            "true" if version >= MzVersion.parse_mz("v0.119.0-dev") else "false"
+        ),
+        "persist_batch_structured_key_lower_len": (
+            "256" if version >= MzVersion.parse_mz("v0.117.0-dev") else "0"
```

Comment:
cc @def- - this was the CI change originally responsible for https://github.com/MaterializeInc/database-issues/issues/8615, which has now been fixed!

I've already got a successful run of nightlies on this branch... anything else you'd like me to check before this goes in?

Reply:
All good, we can reopen in case it reoccurs on main.